components/emoji_picker/emoji_picker_tabs.jsx

```diff
@@ -70,20 +72,16 @@ export default class EmojiPickerTabs extends PureComponent {
                     bottom: this.props.style.bottom,
                     right: this.props.rightOffset,
                 };
```

Comment:
Can we set `this.props.style.top` and `style.left` here instead of using `props.topOffset` and `props.leftOffset`

Reply:
No, we cannot set because the style object in the props is injected by the react-bootstrap Overlay component